code/exceptions/RealMeException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class RealMeException extends Exception
+{
+    const INVALID_IDENTITY_VALUE = 0;
```

Comment:
Would it be better to have subclasses for exception types rather than using constants?